src/main/java/org/example/tms/service/impl/JwtBlacklistServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package org.example.tms.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.example.tms.service.JwtBlacklistService;
+import org.example.tms.util.ConstantUtil;
+import org.springframework.stereotype.Service;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
+
+@Service
+@RequiredArgsConstructor
+public class JwtBlacklistServiceImpl implements JwtBlacklistService {
+
+    private final Map<String, Long> tokenBlacklist = new ConcurrentHashMap<>();
```

Comment:
it s nice for one instance of app, but if u will scale it, u should use something like reddis, just a note